lib/show.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/character'
+
+class Show
+
+  attr_reader :name, :creator, :characters
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+    @salary_total = 0
```

Comment:
don't need this to be an instance variable since you only use it in the method below